interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -608,6 +610,63 @@ def test_skip_undeclared_surf_species(self):
         self.assertEqual(surf.n_reactions, 14)
 
 
+class TestInvalidInput(utilities.CanteraTest):
+
+    _gas_def = """
+            phases:
+            - name: gas
+              thermo: ideal-gas
+              species: [{h2o2.yaml/species: all}]
+              kinetics: gas
+              reactions: all
+              state: {T: 300.0, P: 1 atm, X: {O2: 0.21, N2: 0.79}}
+        """
+
+    def test_failing_convert1(self):
+        # invalid preexponential factor units
+        gas_def = self._gas_def + """
+            reactions:
+            - equation: O + H2 <=> H + OH  # Reaction 3
+              rate-constant: {A: 3.87e+04 cm^6/mol^2/s, b: 2.7, Ea: 6260.0}
+            """
+        check_raises(gas_def, "UnitSystem::convert:", line=12)
+
+    def test_failing_convert2(self):
+        # invalid actiation energy units
+        gas_def = self._gas_def + """
+            reactions:
+            - equation: O + H2 <=> H + OH  # Reaction 3
+              rate-constant: {A: 3.87e+04, b: 2.7, Ea: 6260.0 m}
+            """
+        check_raises(gas_def, "UnitSystem::convertActivationEnergy:", line=12)
+
+    def test_failing_unconfigured1(self):
+        # invalid actiation energy units
```

Comment:
Are these comments correct, in this function and the following ones?

Reply:
no. copy-pasta.